package.json

```diff
@@ -27,31 +27,47 @@
     "Cloud Text-to-Speech API"
   ],
   "scripts": {
-    "test": "nyc mocha",
-    "lint": "eslint '**/*.js'",
-    "samples-test": "cd samples/ && npm link ../ && npm test && cd ../",
-    "system-test": "mocha system-test",
+    "test": "mocha build/test",
```

Comment:
mind making this `c8 mocha build/test`, and adding `c8` as a dependency while removing `nyc`.

We use this for collecting coverage, which gives us valuable insight into what tests are missing.

Reply:
sure, will do, thanks!